attestation-service/src/utils.rs

```diff
@@ -23,12 +24,16 @@ use serde_json::{Map, Value};
 ///     "d": "e"
 /// }
 /// ```
-pub fn flatten_claims(claims: &TeeEvidenceParsedClaim) -> Result<TeeEvidenceParsedClaim> {
+pub fn flatten_claims(
+    tee: kbs_types::Tee,
+    claims: &TeeEvidenceParsedClaim,
+) -> Result<TeeEvidenceParsedClaim> {
     let mut map = Map::new();
+    let tee_type = to_variant_name(&tee)?;
     match claims {
         Value::Object(obj) => {
             for (k, v) in obj {
-                flatten_helper(&mut map, v, k.clone());
+                flatten_helper(&mut map, v, format!("{tee_type}.{}", k.clone()));
```

Comment:
How about using `-` or `_` instead of `.`?

Reply:
In certain specific fields of SGX and TDX, they may already contain `-` or `_`. If we use them as delimiters, it may sometimes cause some confusion. Therefore, I think it is more appropriate for us to use `.` as the delimiter here. For the claims parsed internally by each TEE verifier, they can still use any format they like.